packages/ui/src/organisms/ProductCard/stories/ProductCard.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import type { Story, Meta } from '@storybook/react'
+import type { MouseEvent } from 'react'
+import React from 'react'
+
+// Atoms
+import Price from '../../../atoms/Price'
+import Badge from '../../../atoms/Badge'
+import Button from '../../../atoms/Button'
+// Product Card components
+import ProductCardComponent from '../ProductCard'
+import ProductCardImage from '../ProductCardImage'
+import ProductCardInfo from '../ProductCardInfo'
+import ProductCardHeader from '../ProductCardHeader'
+import ProductCardPrice from '../ProductCardPrice'
+import ProductCardTags from '../ProductCardTags'
+import ProductCardLink from '../ProductCardLink'
+import type { ProductCardProps } from '../ProductCard'
+import mdx from './ProductCard.mdx'
+
+const ProductCardTemplate: Story<ProductCardProps> = ({ testId }) => {
+  function onClick(event: MouseEvent) {
+    event.stopPropagation()
+    event.preventDefault()
+
+    // Do something here
```

Comment:
https://storybook.js.org/docs/react/essentials/actions

Reply:
oh that's nice 💯